Prerequisites.md

```diff
@@ -81,8 +81,15 @@ You can refer to the [Installation Guide](https://github.com/OpenFunction/OpenFu
    strimzi-cluster-operator-687fdd6f77-cwmgm         1/1     Running       0          11m
    ```
 
-   Run the following command to view the metadata for the Kafka cluster.
+   Run the following command to create the utils pod and enter this pod
 
+   ```shell
+   $ kubectl run --rm utils -it --image arunvelsriram/utils bash
+   $ kubectl exec -ti utils -n default -- /bin/sh
```

Comment:
`kubectl exec -ti utils -n default -- /bin/sh` is not necessary, because `kubectl run --rm utils -it --image arunvelsriram/utils bash` already launched and entered the utils pod